app/models/feedback_reporters/feedback_reporter.rb

```diff
@@ -24,31 +24,23 @@ def description
   end
 
   def send_report!
-    # We're sending the XML data via a URL to our Support ticket service. The URL needs to be Percent-encoded so that
-    # everything shows up correctly on the other end. (https://en.wikipedia.org/wiki/Percent-encoding)
+    # We're sending the XML data via a URL to our Support ticket service. The
+    # URL needs to be Percent-encoded so that everything shows up correctly on
+    # the other end. (https://en.wikipedia.org/wiki/Percent-encoding)
     encoded_xml = CGI.escape(xml.to_str)
-    HTTParty.post("#{ArchiveConfig.BUGS_SITE}",
-      body: "&xml=#{encoded_xml}"
-    )
+    HTTParty.post(ArchiveConfig.BUGS_SITE.to_s,
+                  body: "&xml=#{encoded_xml}")
   end
 
   def send_abuse_report!
-    HTTParty.post("#{ArchiveConfig.ABUSE_REPORTS_SITE}/projects/#{project_id}/bugs",
-      headers: {
-        "Content-Type" => "application/xml",
-        "Accept" => "application/xml"
-      },
-      basic_auth: {
-        username: ArchiveConfig.ABUSE_REPORTS_USER,
-        password: ArchiveConfig.ABUSE_REPORTS_PASSWORD
-      },
-      body: xml
-    )
+    encoded_xml = CGI.escape(xml.to_str)
+    HTTParty.post(ArchiveConfig.ABUSE_REPORTS_SITE.to_s,
+                  body: "&xml=#{encoded_xml}")
   end
 
   def xml
-    view = ActionView::Base.new(Rails.root.join("app", "views"))
-    view.assign({report: self})
+    view = ActionView::Base.new(Rails.root.join('app', 'views'))
+    view.assign({ report: self })
     view.render(template: template)
   end
 
```

Comment:
Under here is some code I think can be removed, since we no longer need/have a project_id.